src/modules/auth/component/SignUp/SignUp.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+  Button, Grid,
+  TextField, Typography,
+} from "@mui/material";
+import { styled } from "@mui/material/styles";
+
+const TextFieldStyled = styled(TextField)({
+  marginTop: "51px",
```

Comment:
Avoid using marginTop on a component itself. Try to wrap it in Grid for example or Box and use spacing/margin.
The presentational component like Button, TextField, Typography and etc should NOT use the margins. You can of course use them sometimes, but it is more like an exception to the rule, not the rule.

Especially for the components, that are custom. They should not have the margins/paddings. If you want to make a component universal/generic, you need to avoid using paddings/margins on them. Instead, you import them, wrap the inside the Box or Grid and use margins/paddings on these wrappers

Reply:
done